ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -118,19 +120,29 @@ def gather_missing_licences(data):
        If it doesn't its added to a list of files missing the correct licence."""
     missing_licence = []
     for file_path, extension in data.changed_files:
+        start_comment=False
         with open(file_path, "r") as file:
             for line in file.readlines():  # Read file line by line
                 line = line.strip()  # Remove whitespaces so we can find lines with just comments
-                if extension == "py":
+                if extension in ["py", "msg", "yml", "yaml", "sh"]:
                     if line and len(line) > 1 and line[0] == "#":
                         if "Shadow Robot Company Ltd" in line and "Copyright" in line:
                             if data.current_year not in line:
                                 missing_licence.append(file_path)
-                else:
+                elif extension in ["c", "cpp", "h", "hpp"]:
                     if line and len(line) > 1 and line[0] in ["/","*"]:
```

Comment:
can't this be packed into one if **if line and len(line) > 1 and line[0] in ["/","*", "#"]** as the inside seems to be the same?

Reply:
Yes good point!